import Foundation

enum networkingErrors: Error {
    case customError(Data)
}
